client/src/test/java/nl/altindag/client/ClientConfigShould.java

```diff
@@ -256,6 +260,10 @@ void createJerseyClientWithSecurity() {
         Client client = victim.jerseyClient(sslFactory);
 
         assertThat(client).isNotNull();
+        assertThat(client.getClass().getPackageName())
+        	.as("Jersey JAX-RS implemenatsion is used")
+        	.startsWith("org.glassfish.jersey");
+        
```

Comment:
@Hakky54   I had no better way to ensure that CXF is not substituting jersey in classpath  ..

Reply:
Looks very good!